ts/src/header-validator/source.test.ts

```diff
@@ -1279,6 +1279,69 @@ const testCases: TestCase[] = [
     ],
   },
 
+  {
+    name: 'trigger-state-cardinality-valid',
+    json: `{"destination": "https://a.test"}`,
+    sourceType: SourceType.event,
+    noteInfoGain: true,
```

Comment:
We don't need to cover the info-gain notes in this test, as it's covered elsewhere. Same for `trigger-state-cardinality-invalid`.

Reply:
Done!